collector/config.go

```diff
@@ -62,6 +62,39 @@ type Prometheus struct {
 	MetricsConfig []string `json:"metrics_config"`
 }
 
+type Jolokia struct {
+	//the endpoint to hit to scrape metrics. Eg https://${HOST}/jolokia
+	Endpoint EndpointConfig `json:"endpoint"`
+
+	//holds names of different metrics that can be collected
+	MetricsConfig []JolokiaMetricConfig `json:"metrics_config"`
+
+	//the frequency at which metrics should be collected
+	PollingFrequency string `json:"polling_frequency"`
+}
+
+type JolokiaMetricConfig struct {
+	//the name of the metric Eg "HeapMemoryUsage" or "MyMetricFoo"
+	Name string `json:"name"`
+
+	//the name of the mbean. Eg java.lang:type=Memory
+	MBean string `json:"mbean"`
+	// the attribute from the mbean Eg HeapMemoryUsage
+	Attribute string `json:"attribute"`
```

Comment:
Name is a user generated name for the metric. This is what would show up to the user from the cadvisor results.

Attribute is something specific to the mbean.